recipes/sentry-native/all/conanfile.py

```diff
@@ -130,6 +131,14 @@ def validate(self):
             raise ConanInvalidConfiguration("This version doesn't support ARM compilation")
 
     def build_requirements(self):
+        if hasattr(self, "settings_build") and tools.cross_building(self) and \
+           self.settings.os == "Macos" and self.settings.arch == "armv8":
+            # Workaround for CMake bug with error message:
+            # Attempting to use @rpath without CMAKE_SHARED_LIBRARY_RUNTIME_C_FLAG being
+            # set. This could be because you are using a Mac OS X version less than 10.5
+            # or because CMake's platform configuration is corrupt.
+            # FIXME: Remove once CMake on macOS/M1 CI runners is upgraded.
+            self.build_requires("cmake/3.22.0")
```

Comment:
No, there are 2 issues:
- you need CMake > 3.18 in build requirements of sentry-native, since 0.4.14
- you need a recent CMake version in build requirements of test package if macOS & armv8 due to `CMAKE_SHARED_LIBRARY_RUNTIME_C_FLAG` and `@rpath` bug.